static/src/js/components/SearchAutocomplete/__tests__/SearchAutocomplete.test.jsx

```diff
@@ -78,7 +78,9 @@ describe('SearchAutocomplete', () => {
         await user.type(input, 'test')
       })
 
-      expect(screen.getByText('Loading suggestions...')).toBeInTheDocument()
+      await waitFor(() => {
+        expect(screen.getByText('Loading suggestions...')).toBeInTheDocument()
```

Comment:
I think you can just do a 
```suggestion
        expect(await screen.findByText('Loading suggestions...')).toBeInTheDocument()
```